services_app/build.gradle

```diff
@@ -138,6 +138,8 @@ dependencies {
     implementation group: 'com.google.zxing', name: 'core', version: '3.3.0'
     implementation group: 'com.journeyapps', name: 'zxing-android-embedded', version: '3.5.0'
     implementation 'androidx.test:core:1.4.0'
+    implementation 'androidx.test.espresso:espresso-contrib:3.4.0'
```

Comment:
if this is a test dependency should it not be under testImplementation or androidTestImplementation instead of implementation? Also androidx.test:core should also probably be moved to one or both as implementation is for the actual code not test.

Reply:
Yeah, sorry about that, I was getting the error of checker framework when using the dependency under android test implementation so I put that under implementation. Fixed it by excluding the checker group under android tests.